be/src/vec/exec/scan/vscan_node.h

```diff
@@ -334,6 +334,11 @@ class VScanNode : public ExecNode {
                                                    SlotDescriptor* slot, ColumnValueRange<T>& range,
                                                    PushDownType* pdt);
 
+    template <PrimitiveType T>
+    Status _normalize_match_in_compound_predicate(vectorized::VExpr* expr, VExprContext* expr_ctx,
```

Comment:
_normalize_match_compound_predicate ?

Reply:
`_normalize_match_in_compound_predicate` means match_predicate in compound node tree, `in` is a preposition, not means in_predicate